pkg/embedded/embedded.go

```diff
@@ -2,8 +2,11 @@ package embedded
 
```

Comment:
spoke to @ibuildthecloud , looks like we want to avoid setting up encryption for embedded api server. Lets remove embedded.go changes for now.